azure/ipa.tf

```diff
@@ -463,6 +519,10 @@ aws-fsx-csi-driver:
 metrics-server:
   enabled: false
   EOF
+    ,
+    <<EOT
+${data.github_repository_file.data-pre-reqs-values.content}
+EOT
```

Comment:
Same with this one? For future reference so I know - does indenting format matter at all in these files?

Reply:
WIth the 

<<ZZZ

ZZZ

format, the indentation of the ZZZs doesn't matter. It's essentially a text file. 

Plus, this is exactly what's in AWS at this point.